### ISSUE TITLE
[BITV]: Replace samtically incorrect elements on Artificial Intelligence setting page

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42941]
Radio buttons with one option are not allowed. Please replace notes with `NcNoteCard` component

![Screenshot from 2024-01-18 16-01-48](https://github.com/nextcloud/server/assets/6078378/d031d27a-d48a-452c-8890-7132adfbd13b)

Should be look like:

![image](https://github.com/nextcloud/server/assets/6078378/870b909f-b5d1-4504-a807-62fd897a5e15)

**Comment by JuliaKirschenheuter** [comment]
closing as done
